### ISSUE TITLE
brew link fails because you are trying to replace my curl

### ISSUE THREAD

**Issue by indirect** [shawnxlw/kubernetes-tools#1]
I tried to install kubernetes-tools, but it failed. I expected `brew install shawnxlw/tap/kubernetes-tools` to succeed. Instead, I got this output:

```
$ brew install shawnxlw/tap/kubernetes-tools
==> Installing shawnxlw/tap/kubernetes-tools
==> Downloading https://github.com/shawnxlw/kubernetes-tools/archive/v2.0.0.tar.gz
==> Downloading from https://codeload.github.com/shawnxlw/kubernetes-tools/tar.gz/v2.0.0
######################################################################## 100.0%
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink bin/curl
Target /usr/local/bin/curl
already exists. You may want to remove it:
  rm '/usr/local/bin/curl'

To force the link and overwrite all conflicting files:
  brew link --overwrite kubernetes-tools

To list all files that would be deleted:
  brew link --overwrite --dry-run kubernetes-tools

Possible conflicting files are:
/usr/local/bin/curl
```

**Comment by shawnsw** [comment]
Hi @indirect ,

Thanks for reporting the issue.

This is because the curl for `kcopy` is clashing with your brew installed curl.

I will come up with a fix soon.

**Comment by shawnsw** [comment]
This issue is now resolved in release [v2.0.1](https://github.com/shawnxlw/kubernetes-tools/releases/tag/v2.0.1)
